### ISSUE TITLE
Zone names don’t appear in Home App

### ISSUE THREAD

**Issue by BradJJ2000** [Snakey42069/homebridge-actronair#1]
I have added my system to home bridge. I named the Accessory AirCon and named all 8 zones according to their room. 

In the Home App I put all the tiles into a room named HVAC and selected to display as single tiles so all zones have their own button. It’s all great except that every tile has the title AirCon so I don’t know which zone is which. 

Is there a way to have the Home App display each Zone button with the name of the zone?

**Comment by Snakey42069** [comment]
Hi @BradJJ2000 ,

Thanks for the question. 

Funnily enough I've never used this package, homebridge or HomeKit myself. I made the package as a favour for a friend.

I forwarded your question to my friend and this is the workaround he uses:

_You need to hold down each zone and name it yourself. So you turn on one zone at a time at the panel, and name that corresponding zone on your home app by holding down on the tile and going into the settings._

I hope this helps.

If this doesn't work for you, let me know. I'm happy to take a look at the code if needed.
